app/scripts/views/mixins/modal-settings-panel-mixin.js

```diff
@@ -33,6 +33,10 @@ define(function (require, exports, module) {
       this.navigate('settings/clients');
     },
 
+    _returnToSecurity () {
+      this.navigate('settings/security');
```

Comment:
is there a reason to call this `/security` ? are there plans to put more stuff in this panel in the future? if yes, i guess that's fine, otherwise I would rename the view to fit its title more (Two-step authentication)

Reply:
I used `/security` when we were debating about what to call it, Multi-factor authentication/Two factor authentication and it kinda just stuck with me :)

I don't see a problem using `/two_factor_authentication` now that we have decided to use that name.